spec/requests/home_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Homes", type: :request do
+  describe "GET /home" do
+    it "returns http success" do
+      get "/home"
```

Comment:
I think we can rewrite it into 
```suggestion
      get :index
```

Reply:
Or `get home_index_page` You need to check which one will be working for request tests. :) I do not remember now

Reply:
ok, changed ;)